packages/web-components/src/components/ic-button/ic-button.tsx

```diff
@@ -468,6 +470,70 @@ export class Button {
     return this.variant?.startsWith("icon");
   };
 
+  private arrangeRouterItem = (): void => {
+    const setStyles = (
+      element: HTMLElement | SVGElement,
+      styles: { [key: string]: string }
```

Comment:
This could be a record «Record<string, string>»

Reply:
Thanks for your suggestion @evenstensberg but it looks like this is the type we have used in other similar functions so I think it would be good to keep it consistent